krakenapi.go

```diff
@@ -91,24 +91,31 @@ const (
 	MinimumUSDT = 5.0
 )
 
-// KrakenApi represents a Kraken API Client connection
-type KrakenApi struct {
```

Comment:
I understand that it should be `KrakenAPI` but wouldn't this be a breaking change?

Reply:
fixed

Reply:
fixed